web/src/beta/features/PluginPlayground/Code/utils.ts

```diff
@@ -0,0 +1,30 @@
+const REEARTH_HTML_INJECTION_PATTERN =
+  /reearth\.(ui|modal|popup)\.show\(\s*`([^]*?)`\s*\);/g;
```

Comment:
For these APIs they don't require the content be wrapped by ``, it also possible wrapped by '' or "";
And we also need to consider there could be mulitple API calls exists, the ideal UX is when user's corsor is located within one of the HTML fregment, enable the button, and when user click the button display the edit modal with the certain HTML fregment. You can treat this as next step if don't have good way to achieve that right now.

Reply:
Thank you!
As you pointed out, I’ve added support for backticks and single quotes.
https://github.com/reearth/reearth-visualizer/pull/1236/commits/d3cae8778d9e9c985bdf0aaa4df440d6a57d154a
Considering "multiple reearth.show calls" might take some time to design, so I’ll handle that in a separate pull request.